Sources/TemplateKit/Data/TemplateDataRepresentable.swift

```diff
@@ -89,3 +89,9 @@ extension Date: TemplateDataRepresentable {
         return .double(timeIntervalSince1970)
     }
 }
+
+extension URL: TemplateDataRepresentable {
+    public func convertToTemplateData() throws -> TemplateData {
```

Comment:
All public methods should have doc blocks. For methods that simply conform to a protocol, we usually add something like:
```swift
/// See `TemplateDataRepresentable`.
public func convertToTemplateData() throws -> TemplateData { ... }
```

Reply:
Done